02/components/HeaderCl.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+export default class HeaderCl extends React.Component {
+	render() {
+		this.style = { fontWeight: 700, fontSize: '28px', color: 'rgb(20,130,50)' };
```

Comment:
Jeśli wew. metody to można zrobić po prostu `const styles = { ... }` - będzie lepsze bo mamy zasięg lokalny.